pepper-apis/src/test/java/org/broadinstitute/ddp/route/PatchFormAnswersRouteStandaloneTest.java

```diff
@@ -1865,7 +1865,7 @@ public void testPatch_fileAnswer_createAndUpdate() {
 
         assertEquals(guid, nextGuid);
         answer = (FileAnswer) TransactionWrapper.withTxn(handle ->
-                new AnswerCachedDao(handle).findAnswerByGuid(guid).get());
+                new AnswerCachedDao(handle).findAnswerByGuid(guid).orElse(null));
```

Comment:
Yes. My idea was not to avoid NPE but to handle it in the first assert since it's a more correct way IMHO. Your approach is similar to mine, but if you think its better, I'll convert it to Optional